### ISSUE TITLE
Validation loss is lower than training loss

### ISSUE THREAD

**Issue by olivboom** [msc-acse/acse-9-independent-research-project-OliverJBoom#14]
This shouldn't be the case. Need to investigate

**Comment by olivboom** [comment]
Changed the dataloaders and it now is similar to training loss
